pkg/controller/config/config_controller_test.go

```diff
@@ -0,0 +1,57 @@
+package config
+
+import (
+	"testing"
+
+	mf "github.com/jcrossley3/manifestival"
+	op "github.com/tektoncd/operator/pkg/apis/operator/v1alpha1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+func TestTransformManifest_WithAnnotation(t *testing.T) {
+	resourceWithAnnotation := "../../../deploy/resources/v0.9.2/addons/pipelines/s2i-go/s2i-go-build-deploy.yaml"
+	assert := func(u unstructured.Unstructured, expected string) {
```

Comment:
move this assert function out of `TestTransformManifest_WithAnnotation` as it is reused in next test